server/prisma/schema.prisma

```diff
@@ -11,9 +11,13 @@ datasource db {
 }
 
 model User {
-  id     String  @id @default(uuid())
-  email  String  @unique
-  points Point[]
+  id             String  @id @unique
+  username       String
+  nickname       String
+  email          String  @unique
+  avatar         String
+  toWorkDistance Float   @default(0)
```

Comment:
```suggestion
  distanceToWork Int   @default(2000)
```
I think it will be easier to keep track of total distance, calories and so on if we keep distance in meters rather then km also it is easier to manage int rather then float

Reply:
done